front/components/assistant/conversation/RetrievalAction.tsx

```diff
@@ -16,6 +16,7 @@ import { useState } from "react";
 
 import { classNames } from "@app/lib/utils";
 
+// TODO(2024-01-29 flav) Refactor to use SVG.
 export const PROVIDER_LOGO_PATH: { [provider: string]: string } = {
```

Comment:
We don't have 32x32 logos for new providers, I'll move this part to use proper SVGs.